src/applications/vaos/components/Breadcrumbs.jsx

```diff
@@ -14,6 +14,26 @@ export default function VAOSBreadcrumbs({ children, labelOverride }) {
   const covidLabel = getCovidUrlLabel(location) || null;
   const label = useSelector(state => getUrlLabel(state, location));
   const isCovid = location.pathname.includes('covid-vaccine/');
+
+  const history = useHistory();
+
+  // Only handles breadcrumb isRouterLink:true items, so no routes without that are replaced
+  function handleRouteChange({ detail }) {
```

Comment:
Hmmm can we add some unit tests for this? This seems pretty tricky so we probably want some insurance against future changes.

Reply:
Maybe a cypress e2e test instead of a unit test? Since we want to prevent errors? Or are unit tests good enough?

Reply:
Go for it! I think a e2e to prevent errors would be good, but there are several conditions/branches in `handleRouteChange` that we might want to test more thoroughly so unit tests would be good for that.

Reply:
Sounds good. Will work on that now.

Reply:
Can't get the error to appear on e2e tests without the change, so it's not seeming valuable to add a test for the error/fix.